assets/component-totals.css

```diff
@@ -21,7 +21,11 @@
   margin-left: 2rem;
 }
 
-.totals__subtotal-value {
+.totals__total {
+  margin-top: .5rem;
+}
```

Comment:
I don't think we need this. It applies that margin top even when there are no discounts above it. 

We do have this existing declaration: 
```css
.cart__footer .discounts {
    margin-top: 1rem;
}
```

Which could be changed to `margin-bottom` 👍

Reply:
Then I'd like to see what a designer will think as the spacing between the total and the `taxes and shipping...` is bigger than between the discount and the total 🤔 
![](https://screenshot.click/07-38-an7fa-oza1p.png)